skgstat/tests/test_variogram.py

```diff
@@ -15,10 +15,51 @@
     PLOTLY_FOUND = False
 
 from skgstat import Variogram
+from skgstat import OrdinaryKriging
 from skgstat import estimators
+from skgstat import models
 from skgstat import plotting
 
 
+class TestSpatiallyCorrelatedData(unittest.TestCase):
```

Comment:
NIce. Thanks!
Is your idea to rather replace all the usages of not-correlated samples in the existing unittests with this class or to add additional new tests to the current test suite?
Then we could put the spatially correlated into a test file of its own, as this one is already quite big and confusing.

Reply:
I think given that I use a Variogram & Kriging to generate the data, it's not good to have it in the main class - if _anything_ is broken, every single test might fail in the setUp() function.

But we could of course generate the data some other way, or pre-generate it and store it in a csv file or something.

Reply:
But for now, I just thought sticking new tests in this additional class was the simplest/fastest path to being able to test stuff using correlated data....

Reply:
I also tĺike the idea of rather adding stuff more than replacing the existing. For me, it's also the CSV the way to go, but that is nothing that has to be solved in this PR and is on my agenda anyway. Therefore I'll would suggest of keeping the stuff just like it is and mark this question as resolved. Agree?